### DISCUSSION TITLE
How to configure right-click on taskbar menu?

### DISCUSSION THREAD

**Discussion by PsychoThinker** [home-sweet-gnome/dash-to-panel#1855] [Q&A] [answered]
Could someone explain to me how to configure the menu, which is shown in the picture below (triggered by right-click on taskbar)?

![image](https://user-images.githubusercontent.com/17953552/224558263-5cbb48a9-a07b-4562-94a1-4f8fb78ecd35.png)

For example, I would like to put "Extensions Manager" instead of "Extensions" - is this a dash to panel configuration element?

**Comment by charlesg99** [comment, marked as answer]
This isn't configurable atm, but you could easily change it. Open `~/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com/appIcons.js`, go to line [1810 - 1811](https://github.com/home-sweet-gnome/dash-to-panel/blob/master/appIcons.js#L1809), and replace the title `Extensions` and command `gnome-shell-extension-prefs` to what you want. Pay attention to leave the surrounding single quotes. Save and restart gnome-shell. 

Of course you'll need to make this change again if the extension is updated as it will overwrite this file.

**Reply by PsychoThinker** [reply]
It worked - thank you very much!!

I've been looking for a long time how this configuration can be called and where it can be changed.
